src/components/layout/footer.js

```diff
@@ -25,15 +25,24 @@ const Footer = ({ type, is_ppc, is_ppc_redirect, academy, no_footer_links }) =>
                     )}
                     <DisclaimerSection is_academy={academy} />
                     <Mobile>
-                        <BottomSocialSection type={type} />
-                    </Mobile>
-                    <Desktop>
-                        <NonEU>
+                        <UKEU>
```

Comment:
It would be better if `UKEU` can go above `Mobile` so that as long as it isn't UKEU, it won't render.